tutorials/neurips_challenge_quickstart.md

```diff
@@ -0,0 +1,201 @@
+# NeurIPS 2023 LLM Efficiency Challenge Quickstart Guide
+
+
+
+The [NeurIPS 2023 Efficiency Challenge](https://llm-efficiency-challenge.github.io/) is a competition focused on training **1 LLM for 24 hours on 1 GPU** – the team with the best LLM gets to present their results at NeurIPS 2023.
+
+This quick start guide is a short starter guide illustrating the main steps to get started with Lit-GPT, which was selected as the competition's official starter kit. 
+
+
+
+&nbsp;
+
+## Competition Facts
+
+**Permitted GPUs:**
+
+- 1x A100 (40 GB RAM);
+- 1x RTX 4090 (24 GB RAM).
+
+**Permitted models:**
+
+- All transformer-based LLM base models that are not finetuned yet.
+
+The subset of Lit-GPT models supported in this competition is listed in the Table below.
```

Comment:
```suggestion
The subset of Lit-GPT models supported in this competition is listed in the Table below. These don't include models that have been fine-tuned or otherwise aligned, as per the rules of the challenge.
```

Reply:
nice clarification, thanks!